compair/static/modules/answer/answer-form-partial.html

```diff
@@ -83,6 +83,7 @@ <h2><i class="fa fa-comments"></i> {{assignment.name}}</h2>
 
         <div>
             <label class="required-star" for="importFile">Answer Attachment and/or Text</label>
+            <p ng-show="!answer.file && answer.existingFile" class="alert alert-warning"><strong>Save<span ng-show="answer.draft"> or submit</span> this answer to save changes made to your attachment</strong>, or cancel this window to keep the attachment that appeared when you started editing.</p>
```

Comment:
Should the alert also be displayed when:
- adding an attachment to a draft answer that has no attachment?
- rotating an attached image?